Doctor-Appointment-App/l_static/admin/assets/css/style.css

```diff
@@ -0,0 +1,4143 @@
+/*
+Author       : Dreamguys
+Template Name: Doccure - Bootstrap Admin Template
+Version      : 1.0
+*/
+
+/*============================
+ [Table of CSS]
+
+1. General
+2. Table
+3. Helper Class
+4. Bootstrap Classes
+5. Select2
+6. Nav Tabs
+7. Components
+8. Header
+9. Sidebar
+10. Content
+11. Login
+12. Lock Screen
+13. Notifications
+14. Dashboard
+15. Invoice
+16. Calendar
+17. Inbox
+18. Mail View
+19. Error
+20. Chat
+21. Product
+22. Product Details
+23. Profile
+24. Template Options
+25. Responsive
+
+========================================*/
+
+/*-----------------
+	1. General
+-----------------------*/
+
+@import url('https://fonts.googleapis.com/css?family=Mada:400,500,600,700&amp;display=swap');
+
+@font-face {
+	font-family: 'Material Icons';
+	font-style: normal;
+	font-weight: 400;
+	src: url(../fonts/MaterialIcons-Regular.eot); /* For IE6-8 */
+	src: local('Material Icons'),
+	local('MaterialIcons-Regular'),
+	url(../fonts/MaterialIcons-Regular.html) format('woff2'),
+	url(../fonts/MaterialIcons-Regular.woff) format('woff'),
+	url(../fonts/MaterialIcons-Regular.ttf) format('truetype');
+}
+html {
+	height: 100%;
+}
+body {
+	background-color: #f8f9fa;
+	color: #333;
+	font-family: 'Mada', sans-serif;
+	font-size: 1rem;
+	height: 100%;
+	line-height: 1.5;
+	overflow-x: hidden;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	font-family: 'Mada', sans-serif;
+	margin-top: 0;
+}
+a:hover,
+a:active,
+a:focus {
+	outline: none;
+	text-decoration: none;
+	color: #00d0f1;
+}
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus {
+    -webkit-box-shadow: 0 0 0px 1000px white inset !important;
+}
+.form-control {
+    border: 1px solid #ddd;
+    box-shadow: none;
+    color: #333;
+    font-size: 15px;
+    height: 40px;
+}
+.form-control:focus {
+	border-color: #00d0f1;
+	box-shadow: none;
+	outline: 0 none;
+}
+.form-control.form-control-sm {
+	height: calc(1.5em + .5rem + 2px);
+}
+
+.form-control.form-control-lg {
+	height: calc(1.5em + 1rem + 2px);
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDGCohort2/Team-317-Doctor-Appointment/pullRequest?prid=6207082)